defaults/main.yml

```diff
@@ -1,17 +1,30 @@
 ---
 
-jenkins_version: "2.19.2" # The exact version of jenkins to deploy
+jenkins_version: "2.32.1" # The exact version of jenkins to deploy
 jenkins_url: "http://127.0.0.1" # The url that Jenkins will be accessible on
 jenkins_port: "8080" # The port that Jenkins will listen on
 jenkins_home: /data/jenkins # The directory on the server where the Jenkins configs will live
 
 # If you need to override any java options then do that here.
 jenkins_java_opts: "-Djenkins.install.runSetupWizard=false"
 
+# Configuration files owner and group
+jenkins_config_owner: "ubuntu"
+jenkins_config_group: "ubuntu"
+
 # The locations of the configuration files for jenkins
 jenkins_source_dir_configs: "{{ playbook_dir }}/jenkins-configs"
 jenkins_source_dir_jobs: "{{ jenkins_source_dir_configs }}/jobs"
 
+jenkins_source_config_xml: "{{ jenkins_source_dir_configs }}/config.xml"
+
+jenkins_include_userdata: false
+
+jenkins_userdata: {} # src and dest names
```

Comment:
Well, it's not only configs.
It can be any file related to user's jenkins installation.

Though, I can change it if you think `jenkins_custom_configs` suits more here.